### ISSUE TITLE
ZAC: Document toevoegen / Zaak relateren

### ISSUE THREAD

**Issue by kelvincy** [GemeenteUtrecht/zaakafhandelcomponent#176]
Currently when I try to add a document via `/core/api/documents/upload` with the following data:
```json
{
  "file": "data:text/plain;base64,dGVzdA==",
  "informatieobjecttype": "https://openzaak.utrechtproeftuin.nl/catalogi/api/v1/informatieobjecttypen/58d7a6a9-64ab- 
  45fe-8b3f-03658e08825d",
  "zaak": "https://openzaak.utrechtproeftuin.nl/zaken/api/v1/zaken/e1c2cebd-33af-4d82-9dfb-cabba113ecd4"
}
```
It returns this 400 error: 
`file: ["De verstuurde data was geen bestand. Controleer de encoding type op het formulier."]`

**What data type is accepted here?**

**Comment by sergei-maertens** [comment]
@kelvincy I've been working on the API docs for this endpoint in #174. Short summary: endpoint uses `multipart/form-data` as request content type, and you get JSON back.

**Comment by kelvincy** [comment]
Merged via #173
